src/package.json

```diff
@@ -43,8 +43,8 @@
     "karma-chrome-launcher": "^2.0.0",
     "karma-jasmine": "~0.3.6",
     "karma-phantomjs-launcher": "^1.0.2",
-    "protractor": "5.1.0",
-    "selenium-webdriver": "3.0.1"
+    "protractor": "^5.1.0",
+    "selenium-webdriver": "^3.0.1"
```

Comment:
Just curious, why change this?

Reply:
That should not be here!
I changed it in my local pc to can generate the build and test.

Reply:
But, I still not know why with Jenkins the tests fail